addons/crm/data/crm_lead_demo.xml

```diff
@@ -396,10 +396,7 @@ Andrew</p>]]></field>
             <field name="probability">30.0</field>
             <field name="partner_id" ref="base.res_partner_2"/>
             <field name="email_from">virginie@agrolait.com</field>
-            <field name="country_id" ref="base.be"/>
-            <field name="city">Wavre</field>
-            <field name="street">Rue de Namur 69</field>
-            <field name="phone">+32 10 588 558</field>
+            <field name="phone">+1 10 588 558</field>
```

Comment:
Apparently, the phone number is not in the correct format for the US. As the phone number is incorrect, it negatively impacts the "predictive lead scoring" (PLS) of the lead, leading to a test failure. I think the phone should have the format `+1 XXX XXX XXXX` (where `X` is a digit)

There is a phone number set for `res_partner_2`. If you do not set any phone number at all here, will the system uses that phone number? There is maybe no need to define a phone number for that lead. :smiley:

Reply:
Hello @julien-banken 

https://i.imgur.com/XLrgUZC.png
in res.partner demo data the phone number is like this only  
so do you want me to change the demo data in res partner or just in our module  
like 
`<field name="phone">+1 100 588 5588</field>`

Thanks for the Validation :blush:

Reply:
Sorry my bad 
the demo data is correct but the `crm_case_14`  opp changes the demo data so i had to remove the phone no field from this oppurtinity also

Reply:
And i think we should also change this `Office Design Project` demo data also

Reply:
okay, there are actually other leads assigned to `res_partner_2` (a US company) having a Belgian address (see: `crm_case_26`, `crm_case_27`, `crm_case_28`, etc). We can maybe also remove the custom address and phone number for those records?

I'm scared we are putting our fingers in the gear and that we will have to change everything for the sake of consistency. May I ask you to change that in a separate commit so that we can easily revert the changes if needed?

Reply:
Hello @julien-banken 
Changes are done as per your suggestions,
just waiting for the runbot 
Thanks :blush: